test-integration/operator-tests/src/test/java/org/infinispan/operator/CustomLibsIT.java

```diff
@@ -0,0 +1,107 @@
+package org.infinispan.operator;
+
+import cz.xtf.core.config.XTFConfig;
+import cz.xtf.core.openshift.OpenShift;
+import cz.xtf.core.openshift.OpenShiftWaiters;
+import cz.xtf.core.openshift.OpenShifts;
+import cz.xtf.junit5.annotations.CleanBeforeAll;
+import io.fabric8.kubernetes.api.model.*;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.maven.it.VerificationException;
+import org.apache.maven.it.Verifier;
+import org.assertj.core.api.Assertions;
+import org.infinispan.Infinispan;
+import org.infinispan.Infinispans;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Collections;
+
+@Slf4j
+@CleanBeforeAll
+public class CustomLibsIT {
+    private static final OpenShift openShift = OpenShifts.master();
+    private static final OpenShiftWaiters openShiftWaiters = OpenShiftWaiters.get(openShift, () -> false);
+    private static final Infinispan infinispan = Infinispans.customLibs();
+
+    @BeforeAll
+    static void prepare() throws IOException {
+        preparePVC();
+        preparePod();
+
+        openShiftWaiters.areExactlyNPodsRunning(1).waitFor();
```

Comment:
@Crumby I can't to get it works in the same namespace with operator deployment without your help.
This code just wait for a single Pod (in this case this is operator deployment Pod) without any name or label identification. Could you please to add labels while Pod creation and validate it there by this value.

Also `xtf.cz/keep: "true"` labels validation a bit aggressive, as I expected that nested resources which `ownerReference` has this label defined should not be deleted, but it's not match. Probably you can fix this issue in the `xtf-cz/xtf` project.

Reply:
Waiter should check only the libs loader pod now.

> I expected that nested resources which ownerReference has this label defined should not be deleted, but it's not match

Right, this would be non-trivial though. Since most of the resources are now managed by Operator and tests by themselves are creating only few of them then let the tests manage the resources might be simpler

Reply:
Thanks, @Crumby . Test working now after your fix and #1012